docs/Rules/README.md

```diff
@@ -55,7 +55,7 @@ The PSScriptAnalyzer contains the following rule definitions.
 | [ReservedCmdletChar](./ReservedCmdletChar.md)                                                     | Error       |        Yes         |                 |
 | [ReservedParams](./ReservedParams.md)                                                             | Error       |        Yes         |                 |
 | [ReviewUnusedParameter](./ReviewUnusedParameter.md)                                               | Warning     |        Yes         |                 |
-| [ShouldProcess](./ShouldProcess.md)                                                               | Error       |        Yes         |                 |
+| [ShouldProcess](./ShouldProcess.md)                                                               | Warning     |        Yes         |                 |
```

Comment:
I can confirm this is correct: https://github.com/PowerShell/PSScriptAnalyzer/blob/58c44234d44dfd0db35bb532906963e08fde8621/Rules/UseSupportsShouldProcess.cs#L499